plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -7895,11 +7899,11 @@ msgstr "Il valore ${val} non è consentito dal dizionario dell'elemento ${label}
 
 #: plone.formwidget.recurrence/plone/formwidget/recurrence/browser/i18n.py:139
 msgid "except"
-msgstr ""
+msgstr " eccetto"
```

Comment:
At first I thought that the extra initial space was a typo, but I see you added one also later (see " incluso").

Reply:
Yes, it's to format recurrences output :(